src/App.test.js

```diff
@@ -0,0 +1,41 @@
+import { screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { renderWithProviders } from "./test/utils"
+import App from './App'
+
+it('renders watch later link', () => {
```

Comment:
it will be much more efficient to test component's UI based on snapshots, rather then by checking individual components